<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryReport</class>
 <widget class="QWidget" name="InventoryReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчёт инвентаризация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Тип отчёта</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="2">
       <widget class="QComboBox" name="firm"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="isFirm">
        <property name="text">
         <string>По фирме:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="isDep">
        <property name="text">
         <string>По подразделению:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="isWP">
        <property name="text">
         <string>По рабочему месту/Складу:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="wpName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="selectWPButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="depName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="selectDepButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="givenSubDep">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>С учётом дочерних подразделений</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Вывести отчёт в файл</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Путь:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="fileName">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="selectFileForSave">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hideComposition">
     <property name="text">
      <string>Скрывать состав устройств</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="progressText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Выполнить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/apply/ico/apply_16x16.png</normaloff>:/16x16/apply/ico/apply_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>firm</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>depName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>wpName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectDepButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectWPButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>firm</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>depName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectDepButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>wpName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectWPButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>firm</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>depName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectDepButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>wpName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>selectWPButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>givenSubDep</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWP</sender>
   <signal>clicked(bool)</signal>
   <receiver>givenSubDep</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isFirm</sender>
   <signal>clicked(bool)</signal>
   <receiver>givenSubDep</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
